src/Sylius/Bundle/PromotionBundle/test/app/config/config.yml

```diff
@@ -15,6 +15,18 @@ framework:
         storage_id: session.storage.mock_file
     http_method_override: true
     test: ~
+    messenger:
+        default_bus: sylius.command_bus
```

Comment:
I'm not sure about this approach :/ This is the configuration that would only work in the test application, but still, the subscribed that uses this command bus would not be usable without the CoreBundle - so it's an indirect coupling. How about we use [there](https://github.com/Sylius/Sylius/pull/12990/files#diff-950a994f7a2d29fb277adc619ba01a80a18e79444e7ac0af9eedbdc463c98d28R87) some generic default message bus and override the service definition in CoreBundle to use the `sylius.event_bus`?